catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -201,4 +256,81 @@ public void setParser(Parser parser) {
         this.marshalConfigurator.addProperty(Marshaller.JAXB_FRAGMENT, true);
         this.parser = parser;
     }
+
+    SourceMonitor registrySourceMonitor = new SourceMonitor() {
+
+        @Override
+        public void setAvailable() {
+        }
```

Comment:
I must be using this wrong because I can't get setAvailable() to get triggered...